web/src/data/offers.ts

```diff
@@ -49,17 +50,17 @@ export const createCity = (config: ConnectionConfig, city: CityConfig) =>
 export const removeCity = (config: ConnectionConfig, cityId: number) =>
   Exec(config, "DELETE FROM cities WHERE city_id = ?", cityId);
 
-export const SegmentKinds = ["olc_8", "purchase", "request"] as const;
-export type SegmentKind = typeof SegmentKinds[number];
+export const SegmentKinds = <const>["olc_8", "purchase", "request"];
```

Comment:
we actually use `as const` when needed, you can keep it.

Reply:
I read in best practice to avoid type cast like as const

I knwo even this is kind of typecast but not sure which one among the two is preferred